assets/javascripts/discourse/components/chat-message.js

```diff
@@ -528,11 +539,15 @@ export default Component.extend({
   },
 
   _repositionEmojiPicker(btn, position) {
-    if (!this.element) {
+    const element = document.querySelector(
+      `.chat-message-container[data-id='${this.message.id}']`
+    );
```

Comment:
Maybe this should be extracted to `get element() {}`?

Reply:
ok :+1: